src/main/java/pl/company/foodatu/common/exception/ErrorWrapper.java

```diff
@@ -15,7 +15,7 @@ public ErrorWrapper(List<Error> errors) {
     }
 
     public ErrorWrapper addError(Error error) {
-        if(this.errors == null) {
+        if (this.errors == null) {
```

Comment:
We can add some linter to the project. Are you familiar with any?

Reply:
I use SonarLint extension in InteliJ.